test/vm.install

```diff
@@ -50,14 +50,14 @@ else
     if [ "${branch%fedora*}" != "$branch" ]; then
         mkdir /var/lib/cockpit-test-registry
         chcon -t container_file_t /var/lib/cockpit-test-registry/
-        podman run -d --name ostree-registry --rm -p 5000:5000 -v /var/lib/cockpit-test-registry/:/var/lib/registry localhost/test-registry
+        podman run -d --name ostree-registry -p 5000:5000 -v /var/lib/cockpit-test-registry/:/var/lib/registry localhost/test-registry
         mv /etc/containers/registries.conf /etc/containers/registries.conf.orig
         printf '[registries.insecure]\nregistries = ["localhost:5000"]\n' > /etc/containers/registries.conf
         rpm-ostree compose container-encapsulate \
             --repo=/var/local-repo \
             --label ostree.bootable=true \
             $branch registry:localhost:5000/ostree-oci:cockpit1
-        podman stop ostree-registry
+        podman rm --force --time 0 ostree-registry
```

Comment:
So maybe it was the lack of stop `--force --time 0`? Although one would expect `stop` to be blocking, anyway this works!